### PR TITLE
fix: bug with precommit check using `nvs`

### PR DESCRIPTION
**Pull request by okineadev**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**

In this patch, I added to the `.husky/pre-commit` file an automatic detection of the use of `nvs`, if the user has `nvs` - then the pre-commit script will use the standard version of NodeJS

If the standard version is not installed, this can be done using a simple command:
```bash
nvs link <version>
```

This commit closes the issue with this error - https://github.com/tldr-pages/tldr/issues/12565

### CODE REVIEW

**Review by Teri-anric — APPROVED** [review]
Okey.

**Comment by okineadev** [comment]
@fazlearefin Done!

File: .husky/pre-commit [hunk 1]
```diff
@@ -1 +1,9 @@
-npm test
+#!/usr/bin/env bash
+
+if [ -s "$NVS_HOME/nvs.sh" ]; then
+    . "$NVS_HOME/nvs.sh"
+    # Uses the standard version of NodeJS
+    nvs use
+fi
+
+npm test
```
[the remark this answers is not in the document]
Comment (okineadev) [reply .husky/pre-commit]: Done.

**Comment by gutjuri** [comment]
How would this affect users of nvm that would like to run a nonstandard version of npm?

**Comment by okineadev** [comment]
> How would this affect users of nvm that would like to run a nonstandard version of npm?

This PR contains fixes only for `nvs`, I don't know anything about `nvm` bugs, because I don't use it.

**Comment by gutjuri** [comment]
> > How would this affect users of nvm that would like to run a nonstandard version of npm?
> 
> This PR contains fixes only for `nvs`, I don't know anything about `nvm` bugs, because I don't use it.

Typo :) Of course I meant to ask "How would this affect users of nv**s** that would like to run a nonstandard version of npm?" :)

**Comment by okineadev** [comment]
> > > How would this affect users of nvm that would like to run a nonstandard version of npm?
> > 
> > 
> > This PR contains fixes only for `nvs`, I don't know anything about `nvm` bugs, because I don't use it.
> 
> Typo :) Of course I meant to ask "How would this affect users of nv**s** that would like to run a nonstandard version of npm?" :)

I don't even know if anyone has bugs with this - I'll look into them.

**Comment by okineadev** [comment]
Fixed this error, just reboot the laptop after installing nvs.

**Pull request closed without merging**